arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1,1692 +1,1700 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<array>
```

Comment:
Please revert the indentation changes. Xcode uses tabs in plist files, but the entire file is now indented using spaces. If it were to stay this way, the next time the file is saved by Xcode, it will be re-indented uses tabs, causing almost the entire file to show as modified again, just like it is here.

Reply:
@philium Oops, I thought it would help 😅 It's been reverted!